__tests__/integration.test.js

```diff
@@ -195,4 +195,81 @@ describe('GET /api/articles/:article_id/comments', () => {
             expect(comments).toEqual([])
         })
     })
+})
+
+
+describe('POST /api/articles/:article_id/comments', () => {
+    test('POST 201: endpoint response is the posted comment', () => {
+
+        const newComment = {
+            username: "icellusedkars",
+            body: "the body of the new comment"
+        }
+
+        return request(app)
+        .post("/api/articles/2/comments")
+        .send(newComment)
+        .expect(201)
+        .then(({ body }) => {
+            const { postedComment } = body
+            expect(postedComment).toMatchObject({
```

Comment:
you know exactly what the comment should look like here so I would make the test extra secure by asserting against the values of the posted comment